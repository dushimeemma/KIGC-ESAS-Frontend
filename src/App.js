import React from 'react';
import 'bootstrap/dist/css/bootstrap.css';
import { Provider } from 'react-redux';

import store from './store';
import Routes from './Routes';

const App = () => {
  return (
    <Provider store={store}>
      <Routes />
    </Provider>
  );
};

export default App;
