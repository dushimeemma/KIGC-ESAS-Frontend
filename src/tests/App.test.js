import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('<App/>', () => {
  it('Should render without errors', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
