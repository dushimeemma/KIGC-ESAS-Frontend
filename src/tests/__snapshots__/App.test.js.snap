// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> Should render without errors 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Routes />
</Provider>
`;
